kalite/distributed/static/js/distributed/user/views.js

```diff
@@ -314,4 +314,64 @@ window.UserView = BaseView.extend({
     logout: function() {
         this.model.logout();
     }
-});
\ No newline at end of file
+});
+
+/* This view toggles which navbar items are displayed to each type of user */ 
+window.ToggleNavbarView = BaseView.extend ({
+
+    template: HB.template("user/navigation"),
+
+    initialize: function() {
+
+        _.bindAll(this);        
+        this.listenTo(this.model, "change", this.render);
+        $("topnav").append(this.template());
+
+    },
+
+    render: function() {
+        
+        this.facility_id = {"facility_id": this.model.get("facility_id")};
+        this.$el.html(this.template(this.facility_id));
+        
+        this.userView = new UserView({ model: this.model, el: "#topnav" });
+        $("#topnav").prepend(new AutoCompleteView().$el);
+        
+        // GUEST
```

Comment:
Took your suggestion and it works! But in order to get the handlebars template to read conditions like "is_logged_in" I had to get the view to get that info from the statusModel (renamed this.facility_id to this.user_info):

this.user_info = {
            "facility_id": this.model.get("facility_id"),
            "logged_in": this.model.get("is_logged_in"),
            "admin": this.model.get("is_admin"),
            "super_user": this.model.get("is_django_user")
        };

Is this necessary or is there a way to make the if statements without doing this?